tests/vector/test_info_format_strategies.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from gym.error import InvalidInfoFormat
+from gym.vector.utils import (
+    BraxVecEnvInfoStrategy,
+    ClassicVecEnvInfoStrategy,
+    get_info_strategy,
+)
+
+
+@pytest.mark.parametrize(("num_envs"), [3])
+def test_classic_vec_env_info_strategy(num_envs):
+    infos = ClassicVecEnvInfoStrategy(num_envs)
+    for i in range(num_envs):
+        info = {"example_info": i}
+        infos.add_info(info, i)
+
+    expected_info = [{"example_info": 0}, {"example_info": 1}, {"example_info": 2}]
+    assert expected_info == infos.get_info()
+
+
+@pytest.mark.parametrize(("num_envs"), [3])
+def test_brax_vec_env_info_strategy(num_envs):
+    infos = BraxVecEnvInfoStrategy(num_envs)
+    for i in range(num_envs):
+        info = {"example_info": i}
+        infos.add_info(info, i)
+
+    expected_info = {"example_info": [0, 1, 2]}
+    assert expected_info == infos.get_info()
+
+
+@pytest.mark.parametrize(("num_envs"), [5])
+def test_brax_vec_env_info_strategy_with_nones(num_envs):
+    infos = BraxVecEnvInfoStrategy(num_envs)
+    for i in range(num_envs):
+        if i % 2 == 0:
+            info = {"example_info": i}
+            infos.add_info(info, i)
+
```

Comment:
If we implement numpy arrays, could this test for multiple different dtypes, floats, ints, strings, lists and dicts should be all of the comment ones